pkg/cmd/cli/cmd/set/triggers.go

```diff
@@ -301,7 +310,7 @@ func (o *TriggersOptions) Run() error {
 		return UpdateTriggersForObject(info.Object, updateTriggerFn)
 	})
 	if singleItemImplied && len(patches) == 0 {
-		return fmt.Errorf("%s/%s is not a deployment config or build config", infos[0].Mapping.Resource, infos[0].Name)
+		return fmt.Errorf("%s/%s does not support triggers", infos[0].Mapping.Resource, infos[0].Name)
```

Comment:
Yes, support would be related to resource at some point.  This is just a generic message.